### ISSUE TITLE
MacOS Standalone/Editor does not run on 10.11

### ISSUE THREAD

**Issue by Nolanlemahn** [firebase/quickstart-unity#281]
**Operating system**: macOS
**Operating system version**: 10.11.6

**What steps will reproduce the problem**:
1. Attempt executing code in the editor on listed OS.
2. Attempt executing a standalone build on listed OS.

**What is the expected result?**
1. Code runs normally, as it does on macOS 10.13.x, iOS 11.x, iOS 12.x, Android 8.x, and Windows 10.x.

**What happens instead of that?**
Editor: 
```
Plugins: Couldn't open Assets/Plugins/x86_64/Firebase/FirebaseCppApp-5.2.0.bundle, error: dlopen(Assets/Plugins/x86_64/Firebase/FirebaseCppApp-5.2.0.bundle, 2): Symbol not found: _SecKeyCopyExternalRepresentation
  Referenced from: Assets/Plugins/x86_64/Firebase/FirebaseCppApp-5.2.0.bundle (which was built for Mac OS X 10.12)
  Expected in: /System/Library/Frameworks/Security.framework/Versions/A/Security
 in Assets/Plugins/x86_64/Firebase/FirebaseCppApp-5.2.0.bundle
```
Standalone:
```
Fallback handler could not load library ....app/Contents/Frameworks/MonoEmbedRuntime/osx/FirebaseCppApp-5.2.0
Fallback handler could not load library ....app/Contents/Frameworks/MonoEmbedRuntime/osx/libFirebaseCppApp-5.2.0.dylib
Fallback handler could not load library ....app/Contents/Frameworks/MonoEmbedRuntime/osx/libFirebaseCppApp-5.2.0.so
Fallback handler could not load library ....app/Contents/Frameworks/MonoEmbedRuntime/osx/libFirebaseCppApp-5.2.0.bundle
Fallback handler could not load library ....app/Contents/Frameworks/MonoEmbedRuntime/osx/FirebaseCppApp-5.2.0.dylib
Fallback handler could not load library ....app/Contents/Frameworks/MonoEmbedRuntime/osx/FirebaseCppApp-5.2.0.so
```
(repeating)

**Please provide any other information below, and attach a screenshot if possible.**
So it seems pretty clear to me that the FireBase standalone macOS bundles/plugins were not designed to run on macOS 10.11.*. I disagree with this decision, though, primarily because as far as _Unity 2018.3 is concerned, builds should run on 10.11_.
https://unity3d.com/unity/system-requirements

It's further not clear what other methods/functionality 10.12+ provides that 10.11.x does not, but in my personal opinion, dropping support for 10.11.x just to gain SecKeyCopyExternalRepresentation doesn't seem appropriate.
https://developer.apple.com/documentation/security/1643698-seckeycopyexternalrepresentation
There are (admittedly not as clean) methods for exporting to an external representation (and by extension you then can't use SecKeyCopyPublicKey), but my best guess (obviously I'm not a FireBase developer) is that the 10.12 environment is not critical for what FireBase sets out to do for MacOS.
(I found this - https://forums.developer.apple.com/thread/69135 - I'm not sure if it's useful.)

TL;DR as long as Unity still supports 10.11, I'd love it if FireBase did too, as long as the above reason is why FireBase doesn't support 10.11.

P.S. What's the latest version of the FireBase plugins that are known to support 10.11.x?

**Comment by stewartmiles** [comment]
@Nolanlemahn we hear you, this is not intentional.  AFAIK we have no dependencies upon the latest MacOS SDK, we just need to modify our secure build environment to have the old SDK (it's not just a machine sitting under someone's desk so this isn't a case of just running an installer) and then update our build scripts to use the old SDK.  We have an internal bug tracking this issue.

**Comment by cynthiajoan** [comment]
Update: this is fixed in latest release [6.2.2](https://firebase.google.com/support/release-notes/unity#version_622_-_july_26_2019). Please give it a try, thanks!
